app/forms/reset_password_form.rb

```diff
@@ -69,9 +69,7 @@ def mark_profile_as_password_reset
   end
```

Comment:
I reviewed the code and started testing scenarios today.  Things generally looked good, except for the last step.
` Ensure user is navigated to the ready to verify page`
I was navigated to the `account` page when I logged back in. 🫤 I am unsure why.
I will look at this more on Monday.

Reply:
@shanechesnutt-ft  and I looked at this Monday.  I was navigated to the ready to verify page when I logged back in through the Sinatra OIDC Sample app and selected the Identity Verified service level.  (When I was navigated to the account page, I had logged in directly through idp and no service level was specified.)  As a result, I am satisfied that the flow worked as expected.